packages/contracts/src/deploy-config.ts

```diff
@@ -166,19 +161,49 @@ interface RequiredDeployConfig {
   colosseumProvingTimeout: number
 
   /**
-   * The value used by line 459 of the ZK verifier contract
+   * The value used by line 459 of the ZK verifier contract.
    */
   zkVerifierHashScalar: string
 
   /**
-   * The value used by line 1173 of the ZK verifier contract
+   * The value used by line 1173 of the ZK verifier contract.
    */
   zkVerifierM56Px: string
 
   /**
-   * The value used by line 1173 of the ZK verifier contract
+   * The value used by line 1173 of the ZK verifier contract.
    */
   zkVerifierM56Py: string
+
+  /**
+   * The number of confirmations required to execute a transaction.
+   */
+  securityCouncilNumConfirmationRequired: number
```

Comment:
Could you add `securityCouncilTokenOwner` to `DeployConfig`? It is just a string.